### PR TITLE
[CSL-737] Add few fixes to launcher params

### PR DESCRIPTION
**Pull request by volhovM**
cc @domenkozar

### CODE REVIEW

File: installers/WindowsInstaller.hs [hunk 1]
```diff
@@ -1,32 +1,34 @@
...
-        "--logs-prefix", "\"%APPDATA%\\Daedalus\\Logs\"",
-        "--db-path", "\"%APPDATA%\\Daedalus\\DB-0.2\"",
-        "--wallet-db-path", "\"%APPDATA%\\Daedalus\\Wallet-0.2\"",
-        "--wallet"
-        ] <> ("--peer" : (L.intersperse "--peer" ipdht))
+    [ "--node \"%PROGRAMFILES%\\Daedalus\\cardano-node.exe\""
+    , "--node-log-path", "\"%APPDATA%\\Daedalus\\Logs\\cardano-node.log\""
+    , "--wallet \"%PROGRAMFILES%\\Daedalus\\Daedalus.exe\""
+    , "--updater \"" <> updaterPath <> "\""
+    , "--node-log-config", "log-config-prod.yaml" -- should it be just here, not in programfiles/appdata?
```
Comment (volhovM) [inline installers/WindowsInstaller.hs]: @domenkozar where should it point to? I think we should put a full path here, not a relative (from what dir?).

Comment (domenkozar) [inline installers/WindowsInstaller.hs]: It's fine here, current directory points to `C:\Program Files\Daedalus`.

File: installers/WindowsInstaller.hs [hunk 2]
```diff
@@ -1,32 +1,35 @@
...
-        "--keyfile", "\"%APPDATA%\\Daedalus\\Secrets\\secret.key\"",
-        "--logs-prefix", "\"%APPDATA%\\Daedalus\\Logs\"",
-        "--db-path", "\"%APPDATA%\\Daedalus\\DB-0.2\"",
-        "--wallet-db-path", "\"%APPDATA%\\Daedalus\\Wallet-0.2\"",
-        "--wallet"
-        ] <> ("--peer" : (L.intersperse "--peer" ipdht))
+    [ "--node \"%PROGRAMFILES%\\Daedalus\\cardano-node.exe\""
+    , "--node-log-path", "\"%APPDATA%\\Daedalus\\Logs\\cardano-node.log\""
+    , "--wallet \"%PROGRAMFILES%\\Daedalus\\Daedalus.exe\""
+    , "--updater \"" <> updaterPath <> "\""
```
Comment (domenkozar) [inline installers/WindowsInstaller.hs]: Does this compile? Where does `updaterPath` come from?

File: installers/WindowsInstaller.hs [hunk 3]
```diff
@@ -1,32 +1,35 @@
...
-        "--db-path", "\"%APPDATA%\\Daedalus\\DB-0.2\"",
-        "--wallet-db-path", "\"%APPDATA%\\Daedalus\\Wallet-0.2\"",
-        "--wallet"
-        ] <> ("--peer" : (L.intersperse "--peer" ipdht))
+    [ "--node \"%PROGRAMFILES%\\Daedalus\\cardano-node.exe\""
+    , "--node-log-path", "\"%APPDATA%\\Daedalus\\Logs\\cardano-node.log\""
+    , "--wallet \"%PROGRAMFILES%\\Daedalus\\Daedalus.exe\""
+    , "--updater \"" <> installerPath <> "\""
+    , "--node-log-config", "log-config-prod.yaml" -- should it be just here, not in programfiles/appdata?
+    , "--node-log-path", "\"%APPDATA%\\Daedalus\\Logs\\nodeLogStdout.log\""
```
Comment (jmitchell) [inline installers/WindowsInstaller.hs]: This should be removed.

Comment (volhovM) [inline installers/WindowsInstaller.hs]: For what reason?

Comment (domenkozar) [inline installers/WindowsInstaller.hs]: It duplicates an arg 4 lines above.

**Comment by jmitchell** [comment]
@volhovM: There's pretty good evidence this change caused a regression, although I haven't yet found a root cause. Using the latest installers I get the following error when I double click the Daedalus desktop shortcut.

daef-12 cardano-launcher exe error2

I've confirmed Appveyor build 0.1.332 (a10383e0) doesn't exhibit this error, however build 0.1.341 (34816aea) does. Both those builds used artifacts from the same Daedalus build (0.1.2908), so I suspect something in these changes is the cause.

Initially I suspected non-ASCII characters in paths were causing it because we saw a similar error in DAEF-12 (user's with non-ASCII usernames have non-ASCII `%APPDATA%` paths; special care must be taken when printing unicode from Haskell in Windows).

**Comment by volhovM** [comment]
I don't think you're precisely right. Usually when we're getting problems with unicode output on windows, the error is the following: `hPutChar: invalid argument (invalid character)`, but i've checked it once again and removed all unicode chars from string literals in csl (just in case). This particular problem might be caused by wrong resourse allocation/releasing, but i've never encountered it. Please share logs if you have any.

**Comment by jmitchell** [comment]
We discovered the extraneous `--node-log-path` was causing the error.

There was a subsequent unicode error, but it's also been resolved. The new `--updater` parameter led cardano-launcher.exe could to sometimes print a path with non-ASCII characters. The offending line has been commented out with a reference to DAEF-12.

Things are looking good now. Thanks!

**Pull request merged**